core/src/main/java/org/apache/calcite/adapter/jdbc/JdbcRules.java

```diff
@@ -765,10 +765,11 @@ public RelNode convert(RelNode rel) {
      */
     public RelNode convert(Sort sort, boolean convertInputTraits) {
       final RelTraitSet traitSet = sort.getTraitSet().replace(out);
+      final RelTraitSet inputTraitSet = sort.getInput().getTraitSet().replace(out);
 
```

Comment:
Move this line below `if (convertInputTraits)` where it is actually used.

Reply:
Thank you, I will update it.